server/server.rb

```diff
@@ -15,6 +15,7 @@
 class Server < Roda
   VERSION = File.read('./VERSION').strip
 
+  use Rack::Deflater, include: %w(application/json application/vnd.api+json application/yaml)
```

Comment:
Of these the master will probably just return `application/json`.

(Maybe the streaming endpoints could set content type to `application/json+streaming` and `Rack::Deflater` would then skip zipping those?)